### PR TITLE
Add App and Io code for AIRS

### PR DESCRIPTION
**Pull request by pasitlaothamatas**
### Summary

- Added App and IO code to broadcast AIR states

### Testing Done

- Added unit tests for BMS-38 to check that AIRs' statuses can be broadcasted at 100Hz in all states

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [ ] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [ ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: boards/BMS/Src/Io/Io_Airs.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+           HAL_GPIO_ReadPin(AIR_POWER_STATUS_GPIO_Port, AIR_POWER_STATUS_Pin);
+}
+
+bool Io_Airs_IsAirNegativeOn(void)
+{
+    return Io_Airs_IsAirPositiveOn() ||
+           (HAL_GPIO_ReadPin(
+                AIR_POWER_STATUS_GPIO_Port, AIR_POWER_STATUS_Pin) &&
+            !HAL_GPIO_ReadPin(
+                AIR_TOTAL_ISENSE_GPIO_Port, AIR_TOTAL_ISENSE_Pin));
```
Comment (alxshev) [inline boards/BMS/Src/Io/Io_Airs.c:28]:
```suggestion
    return HAL_GPIO_ReadPin(AIR_POWER_STATUS_GPIO_Port, AIR_POWER_STATUS_Pin);
```

File: boards/BMS/Inc/Io/Io_Airs.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <stdbool.h>
+
+/**
+ * Check if AIR+ is on
+ * @return true if AIR+ is on. Else, false
+ */
+bool Io_Airs_IsAirPositiveOn(void);
```
Comment (alxshev) [inline boards/BMS/Inc/Io/Io_Airs.h:9]: I'd put the AIR- first so it's consistent with the App code

File: boards/BMS/Src/Io/Io_Airs.c (hunk 1 above)
Comment (pasitlaothamatas) [reply boards/BMS/Src/Io/Io_Airs.c:28]: good point

File: boards/BMS/Inc/Io/Io_Airs.h (hunk 2 above)
Comment (pasitlaothamatas) [reply boards/BMS/Inc/Io/Io_Airs.h]: Sounds good. I applied the changes

**Comment by pasitlaothamatas** [comment]
@alxshev Addressed your suggestions 🚀

**Review by alxshev — APPROVED** [review]
wonderful

**Pull request merged**